extras/book-theme.css

```
@@ -42,6 +42,14 @@
   --md-accent-fg-color:         #42b983;
 }
 
+/* Prefer Korean glyph forms only while viewing the Korean edition. Keeping
+   this override language-scoped avoids changing Chinese glyphs site-wide. */
+html[lang="ko"] {
+  --md-text-font: "Noto Sans CJK KR", "Apple SD Gothic Neo", "Malgun Gothic",
+                  "Noto Sans SC", "PingFang SC", -apple-system,
+                  BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif;
```

Stylelint (17.14.1)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)